contents/javascript/VueJs/Intro.md

```diff
@@ -0,0 +1,133 @@
+# VueJs
+
+Author: [Lu Lechuan](https://github.com/LuLechuan)
+
+Vue.js is an open-source progressive [JavaScript framework](https://en.wikipedia.org/wiki/JavaScript_framework) for building user interfaces. It is designed to improve code quality and maintainablity. Simply put, it is used as a tool to write JavaScript fast and easy.
+
+## Advantages of VueJs
+
+VueJs is an approachable, versatile and performant framework that helps to create a more maintainable and testable database.
+
+* Approachable: VueJs is very easy to learn. Compared to other framework such as Augular, Vue is simple in term of API and design. Learning enough to build non-trivial applications typically takes less than a day.
+
+* Progressive: Vue is designed from the ground up to be incrementally adoptable. The core library is focused on the view layer only, and is easy to pick up and integrate with other libraries or existing projects. This means that if you have a large application, you can plug Vue into just a part of your application without disturbing the other components. A quote from Evan You - the owner of Vue is as follow:
```

Comment:
Sometimes you use Vue, sometimes VueJs. Perhaps it's better to stick to one?

Reply:
Ok I will include the short cut. But I will change all the Vue to VueJs for consistency, except for "Vue components" or "Vue instance"